### ISSUE TITLE
FEATURE REQUEST

### ISSUE THREAD

**Issue by Qazeer** [fireeye/commando-vm#65]
**Would you like have a tool or script added? Please include a link**
Configure the winrm service (Windows built-in)  for CredSSP usage and credentials delegation.

**Is your feature request related to a problem? Please describe.**
The current configuration of the winrm service does not support use of CredSSP authentication with credentials delegation, needed to solve the "double hop" authentication problem that may occur whenever pivoting from remote targets using winrm.
 
**If related to a problem, describe the solution you'd like**
The following steps can be used to configure the winrm service to trust all machines for credentials delegation:

```
winrm quickconfig
set-item wsman:localhost\client\trustedhosts -value *
Enable-WSManCredSSP -Role "Client" -DelegateComputer "*"

Start gpedit.msc
-> "Local Computer Policy" -> "Computer Configuration" -> "Administrative Templates" -> "System" -> "Credential Delegation"
-> In the "Settings" pane, "Allow Delegating Fresh Credentials with NTLM-only Server Authentication". -> "Enabled"
-> And in the "Options" area, "Show" -> "Value" = WSMAN/*
-> "Concatenate OS defaults with input above" checked
```

Thanks for the good work and useful VM!

**Comment by day1player** [comment]
@Qazeer Thank you for the suggestion! I will see what I can do to get this configuration change made.

**Comment by day1player** [comment]
@quazeer this should be fixed now with the 1.3 release. If you end up reinstalling the VM, let us know if you continue to have issues. Thanks!
